src/e2e/java/uk/gov/hmcts/reform/sscs/functional/handlers/postpone/PostponeHearingHandlerTest.java

```diff
@@ -1,42 +1,56 @@
 package uk.gov.hmcts.reform.sscs.functional.handlers.postpone;
 
 import static io.restassured.RestAssured.given;
-import static org.junit.Assert.assertEquals;
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.YesNo.NO;
 import static uk.gov.hmcts.reform.sscs.util.SyaJsonMessageSerializer.POSTPONE_HEARING_CALLBACK_CCD;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.restassured.common.mapper.TypeRef;
 import io.restassured.http.ContentType;
+import org.apache.http.HttpStatus;
 import org.junit.Test;
+import org.junit.jupiter.api.DisplayName;
 import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.http.HttpStatus;
 import org.springframework.test.context.TestPropertySource;
 import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.hmcts.reform.sscs.ccd.domain.DwpState;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
 import uk.gov.hmcts.reform.sscs.functional.handlers.BaseHandler;
 
 @RunWith(SpringRunner.class)
 @TestPropertySource(locations = "classpath:config/application_e2e.properties")
 @SpringBootTest
 public class PostponeHearingHandlerTest extends BaseHandler {
 
+    @Autowired
+    private ObjectMapper mapper;
+
+    @DisplayName("When a hearing postpone callback is made, the correct fields are changed")
```

Comment:
Vague description. What are the "correct fields"?

Might want to split this test into multiple ones, one for each field you want to check.